### PR TITLE
update editor action schema version

### PR DESCRIPTION
**Pull request by xavluiz**
update editor action schema version for scroll events

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,11 +23,12 @@
...
   "dependencies": {
-    "lodash": "^4.17.19",
     "@types/node": "^14.0.13",
     "axios": "^0.19.2",
     "libsodium": "^0.7.6",
     "libsodium-wrappers": "^0.7.6",
-    "snowplow-tracker": "^0.3.0"
+    "lodash": "^4.17.19",
+    "snowplow-tracker": "^0.3.0",
+    "swdc-vscode": "/Users/xavierluiz/software/swdc-vscode"
```
Comment (mmclead) [inline package.json:32]: That's going to cause issues yes?

File: src/utils/http.ts [hunk 2]
```diff
@@ -1,28 +1,33 @@
 import axios from "axios";
 
-let baseURL = "";
+// build the axios api base url
+let beApi: any = {};
```
Comment (mmclead) [inline src/utils/http.ts:4]: What does "be" stand for?

File: src/utils/http.ts [hunk 3]
```diff
@@ -1,28 +1,33 @@
...
 
 export function setBaseUrl(url: string) {
-  baseURL = url;
+  beApi = axios.create({
+    baseURL: url,
+    timeout: 15000 // timeout so we're not getting ECONNRESET
```
Comment (mmclead) [inline src/utils/http.ts:9]: That's pretty long. Is that a reasonable timeout?

File: src/utils/http.ts (hunk 2 above)
Comment (xavluiz) [reply src/utils/http.ts:4]: backend, it can be any variable name.

File: src/utils/http.ts (hunk 3 above)
Comment (xavluiz) [reply src/utils/http.ts:9]: yeah, its a bit long. we can shorten it. what do you think 5 seconds?

**Pull request merged**